trending_data/src/main/webapp/script.js

```diff
@@ -174,21 +173,95 @@ function createIframeById(video, id) {
  * @return {HTMLElement}
  */
 function getVideosNode(videos) {
-  const div = document.createElement('div');
-  div.className = 'video-list';
+  const videoList = document.createElement('div');
+  videoList.className = 'video-list';
 
   /** Add title to list of videos to make it clear
     what data is being displayed */
   const ytTitle = document.createElement('h2');
   ytTitle.innerText = 'YouTube videos trending in ' +
     windowsHandler.getCountryName();
-  div.appendChild(ytTitle);
+  videoList.appendChild(ytTitle);
 
   for (let i = 0; i < videos.length; i++) {
-    const currentVideo = createIframeById(videos[i], i);
-    div.appendChild(currentVideo);
+    videoList.appendChild(createVideoContainer(videos[i], i));
   }
-  return div;
+  return videoList;
+}
+
+/**
+* Creates video container for one video.
+* Contains the video (iframe) and a
+* metadta section
+* @param {object} video id of video on Youtube
+* @param {number} id id that will have iframe element
+* @return {HTMLElement}
+*/
+function createVideoContainer(video, id) {
+  const videoContainer = document.createElement('div');
+  videoContainer.className = 'video-container';
+
+  const iframe = createIframeById(video, id);
+  const metadata = createMetadataContainer(video);
+
+  videoContainer.appendChild(iframe);
+  videoContainer.appendChild(metadata);
+  return videoContainer;
+}
+
+/**
+* Creates the video metadata section with
+* channel name, view and like count
+* @param {object} video id of video on Youtube
+* @return {HTMLElement}
+*/
+function createMetadataContainer(video) {
+  const metadata = document.createElement('div');
+  metadata.className = 'yt-metadata';
+
+  const channelNameDiv = createMetadataBit('Channel', video.channelName);
+  const viewCountDiv = createMetadataBit('Views',
+      numberWithCommas(video.viewCount));
+  const likeCountDiv = createMetadataBit('Likes',
+      numberWithCommas(video.likeCount));
+
+  metadata.appendChild(channelNameDiv);
+  metadata.appendChild(viewCountDiv); ;
+  metadata.appendChild(likeCountDiv);
+
+  return metadata;
+}
+
+/**
+* Creates a piece of metadata: comprised of a tag
+* and the corresponding value
+* e.g. channel (tag): good music (value)
+* @param {String} tag what metadata is displayed
+* @param {object} value value of data displayed
+* @return {HTMLElement}
+*/
+function createMetadataBit(tag, value) {
```

Comment:
I was a bit confused by the term 'Bit' here at first. Maybe 'section' or 'field' might be clearer?